app/models/profile_rule_group.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Join table to be able to have a has-many-belongs-to-many relation between
+# Profile and RuleGroup
+class ProfileRuleGroup < ProfileLink
+  belongs_to :profile
+  belongs_to :entity, :class_name => 'RuleGroup', :foreign_key => :entity_id
+
+  validates :profile, presence: true
+  validates :entity_id, presence: true, uniqueness: { scope: :profile }
+end
```

Comment:
Missing newline + please change it analogously to the `ProfileRule` as I suggested above.